### PR TITLE
moveMouse refactor to use sendInput on Windows (#26)

### PR DESCRIPTION
**Pull request by Reiss-Cashmore**
Hi,

This PR should nicely refactor the moveMouse method
Ref: https://github.com/nut-tree/libnut/issues/26

I believe everything here is inline with the Microsoft documentation and guidelines here
https://docs.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-mouseinput

There might still be some flags worth exploring and dwExtraInfo but I am not too sure

Not sure if you want the helper methods for converting the points to absolute coords to live within mouse.c

### CODE REVIEW

**Comment by Reiss-Cashmore** [comment]
By the looks of it, it failed because it's worried about some type coercion and loss of data through the casting. I'll need to probably read up a bit as I really don't know C++ very well

File: src/win32/mouse.c [hunk 1]
```diff
@@ -20,13 +20,30 @@
 	: ((button) == RIGHT_BUTTON ? MOUSEEVENTF_RIGHTDOWN \
 	: MOUSEEVENTF_MIDDLEDOWN))
 
+int CalculateAbsoluteCoordinateX(int x) {
+	MMSize displaySize = getMainDisplaySize();
+	return (x * 65536) / displaySize.width;
```
Comment (s1hofmann) [inline src/win32/mouse.c:25]: **Issue:** Please introduce a constant holding `65536` and consider an explanatory comment linking to the docs. Otherwise it's really hard to guess what this magic number is used for.

File: src/win32/mouse.c [hunk 2]
```diff
@@ -20,13 +20,30 @@
...
+}
+
+int CalculateAbsoluteCoordinateY(int y) {
+	MMSize displaySize = getMainDisplaySize();
+	return (y * 65536) / displaySize.height;
+}
+
```
Comment (s1hofmann) [inline src/win32/mouse.c:32]: **Suggestion:** `moveMouse` receives an `MMPoint` struct, so we could merge these to methods to receive and return an `MMPoint` as well. This way we would also only have to call `getMainDisplaySize` once.

File: src/win32/mouse.c (hunk 1 above)
Comment (s1hofmann) [inline src/win32/mouse.c:25]: **Nitpick:** I would rephrase the calculation to `(x / displaySize.width) * 65536` and `(y / displaySize.height) * 65536`, but that's just my personal tase and due to how I read the equation.
